src/py42/sdk/__init__.py

```diff
@@ -72,14 +74,17 @@ def from_local_account(cls, host_address, username, password):
                 console.us.code42.com
             username (str): The username of the authenticating account.
             password (str): The password of the authenticating account.
-
+            totp (str): The time-based one-time password of the authenticating account. Include only
```

Comment:
The type of `totp` should be `(str, optional)` and the doc should end with the sentence ` Defaults to None.`

Reply:
Updated!